idunn/blocks/commercial.py

```diff
@@ -0,0 +1,24 @@
+from .base import BaseBlock
+from typing import Literal, Optional
+
+
+class CommercialBlock(BaseBlock):
+    type: Literal["commercial"] = "commercial"
+    booking_url: Optional[str]
+    appointment_url: Optional[str]
+    order_url: Optional[str]
+
+    @classmethod
+    def from_es(cls, place, lang):
+        args = {
+            "booking_url": place.get_booking_url(),
+            "appointment_url": place.get_appointment_url(),
+            "order_url": place.get_appointment_url(),
+        }
+
+        args = {key: val for key, val in args.items() if val}
+
+        if not args:
+            return None
```

Comment:
Is this transformation only useful to check that a non-empty value exists?  
If so, it could be more explicit to write:
```suggestion
        if not any(args.values()):
            return None
```

Reply:
:+1: 
I was uncomfortable with all the `none` fields this block could add an hoped that [`response_model_exclude_unset`](https://fastapi.tiangolo.com/tutorial/response-model/#use-the-response_model_exclude_unset-parameter) would help me here, but it's not even designed to solve this problem...